core/storage/user/gae_models.py

```diff
@@ -63,6 +63,13 @@ class UserSettingsModel(base_models.BaseModel):
         choices=[
             constants.DASHBOARD_TYPE_LEARNER,
             constants.DASHBOARD_TYPE_CREATOR])
+    # The preferred dashboard display preference.
+    creator_dashboard_display = ndb.StringProperty(
+        default=feconf.CREATOR_DASHBOARD_DISPLAY_PREF_CARD,
+        indexed=False,
```

Comment:
indexed=True. Might be useful for debugging later.

Reply:
Done